tests/test_geometry.py

```diff
@@ -51,3 +55,240 @@ def test_check_if_geometry_is_valid():
     mc.check_if_geometry_is_valid(1)
 
     mc.set_variable("Slot_Depth", save_slot_depth)
+
+
+def test_set_adaptive_parameter_value():
+    mc.set_adaptive_parameter_value("test_parameter", 100)
```

Comment:
Can we check the dataStore entry with get_array_variable and assert these are the same.

Also would be good to have 2 cases to test overwriting and adding a new var.